Added new property `members_array` for group & groups resources. (#5479)
chef/inspec: docs-chef-io/content/inspec/resources/groups.md

    its('members') { should include 'root' }
    its('members') { should include 'Administrator' }

> where `members` returns
> - Array of group members for **Windows Platform**.
> Example: `["member1", "member2"]`
> - Single element array with CSV formatted string of group members for **Non-Windows Platforms**.
> Example: `["member1,member2"]`

Reviewer: ```suggestion
where `members` returns:

- an array of group members for **Windows Platform**.

   Example: `["member1", "member2"]`

- a CSV formatted string of group members for **Non-Windows Platforms**.

   Example: `"member1,member2"`
```

Author: @IanMadd Is there any reason behind the edit on line 83 and 85 because it goes against the fact that it returns a single element array of format `["member1,member2"]` ?
We should not let this information change I feel because it does return an array in groups resource on line 83 & 85.

Reviewer 2: If I am not wrong we are introducing new property so that it returns the array of members regardless of any OS, so as per me we should return the result in the format of ["member1", "member2"] regardless of any OS and if they are different set of members and not the ["member1, member2"]..

Author: @Vasu1105 Yes this document change above is for already existing property `members` to clarify more on the way it is already working for different platforms. For `members_array` property the documents always states to return an array irrespective of the platform.

Reviewer: How about this instead:

`a single element array that contains a CSV string of group members for **Non-Windows Platforms**.`

### members_array

> The `members_array` property tests the group members just like `members` property.
> But the value returned by this property is always an array of group members.

Reviewer: ```suggestion
The `members_array` property tests the group members just like the `members` property,
but the value returned by this property is always an array of group members.
```
